script/Ciel/Day 14/14_01B_03_CIEL14B_4_1E.txt

```diff
@@ -371,20 +444,22 @@ My eyes become stunned at the change in the Luminary which I should have been us
 [sha:edd7fdaf18bb72ab4b35df573aa81c526ee210a2]{
 -- Page 133, Offset 40635. Mods: @n.
 -- ロアの住み処ごと、
-I'll trample on Roa's den
+I'll go trample Roa's den―――
 }
 [sha:ebc20dfe1795a111f54dbbe796ba6979abc05b70]{
 -- Page 133, Offset 40636.
 -- アナタたちの思い出を踏み潰してあげる”」
-along with all your memories."
+Along with all the memories you two shared."
 }
 [sha:cf73babc77c7a44d28dc751a35590e1ffe5ed284]{
 -- Page 134, Offset 40637.
 -- 悪態も忘れて、呆然と見送るしかない。
-I find myself doing nothing but look in awe, forgetting about her foul mouth in the process.
+Her taunt sails right over my head.
+ I'm too dumbstruck to do anything but stare.
 }
 [sha:885b5f3412305b6a98d3f2deda0b3521827a5fd5]{
 -- Page 134, Offset 40638.
 -- それは“動かない”という唯一の欠点を気分で取りやめた、この地上で最も尊大で我が儘な、吸血姫の誕生だった。
-It was the birth of the most arrogant and selfish vampiress on the planet, who easily disregarded her only flaw of being 'immobile' at her own whim.
+I'm witnessing the birth of the most arrogant and selfish vampiress on the planet;
+ one who cast off her immobility――her only weak point――simply because she felt like it.
```

Comment:
small tweaks
>I'm witnessing the birth of the most haughty and selfish vampiress on the planet;
>one who cast off her only weak point――her immobility――simply because she felt like it.